parts/appendices/A.fodt

```diff
@@ -6797,9 +6797,8 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
       <table:covered-table-cell/>
       <table:covered-table-cell/>
       <table:covered-table-cell/>
-      <table:table-cell table:style-name="Table35.H5" office:value-type="string">
-       <text:p text:style-name="P2480"/>
-      </table:table-cell>
+      <table:table-cell table:style-name="Table38.H21" office:value-type="string">
+       <text:p text:style-name="P2480"/>      </table:table-cell>
```

Comment:
Strange, it seems like the script removes a  line break, @hakonhagland

Reply:
@blattms Yes that seems strange.. Let me check what is going on.

Reply:
Think I found the issue, see #248.